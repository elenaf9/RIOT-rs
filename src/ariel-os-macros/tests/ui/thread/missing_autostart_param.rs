// #![no_std]
#![no_main]

// FAIL: the `autostart` parameter is mandatory
#[ariel_os::thread]
fn main() {}
