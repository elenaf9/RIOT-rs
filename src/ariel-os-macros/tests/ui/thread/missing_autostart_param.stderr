error: custom attribute panicked
 --> tests/ui/thread/missing_autostart_param.rs:5:1
  |
5 | #[ariel_os::thread]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = help: message: the `autostart` parameter must be provided
